pegasus/src/curriculum_course.rb

```diff
@@ -1,8 +1,7 @@
-Unit = Struct.new(:number, :title, :description)
-
-Lesson = Struct.new(:number, :url, :title, :description, :path)
-
 class CurriculumCourse
```

Comment:
What does CurriculumCourse do?

Reply:
I don't have context here but it looks like it provides some metadata about courses to various places in Pegasus. I had to change this because the Lesson defined here were conflicting with the model definition.